libcxx/test/std/atomics/atomics.types.generic/atomics.types.float/exchange.pass.cpp

```diff
@@ -8,7 +8,9 @@
 // UNSUPPORTED: c++03, c++11, c++14, c++17
 // UNSUPPORTED: target={{.+}}-windows-gnu
 // Clang's support for atomic operations on long double is broken. See https://github.com/llvm/llvm-project/issues/72893
-// XFAIL: tsan, msan
+// XFAIL: tsan
+// Hangs with msan.
+// UNSUPPORTED: msan
```

Comment:
do you know why it is hanging on your CI? It was fine on our msan CI. It XFAIL because `long double` of x87_fp80 format has padding bits and at the moment, there is no way for clang to initialise those padding bits

Reply:
Not yet, now I want recover bots and investigate later, if there is an interest in support of these test.

Reply:
Is x87_fp80 about msan? Msan, I guess, can just not take them into account?
I didn't look into details of float atomics, but if the type is visible to msan, it should be possible.

Any idea for tsan?

Reply:
yes it is about msan .

TSAN is because of the GitHub issue in the comments. basically according to the discussion it is a problem in TSAN itself where it does not intercept any calls into libatomic